src/ConfigProvider.php

```diff
@@ -18,19 +18,20 @@ public function __invoke() : array
     {
         return [
             'dependencies' => $this->getDependencies(),
-            'templates' => $this->getTemplates(),
+            'templates'    => $this->getTemplates(),
```

Comment:
I'm not a big fan of this kind of updates/alignments. it looks nice as long as we need to add new value with longer key and then diff is unclear if we keep the alignment.

Reply:
We need a coding style rule for this. On some places it's done, other places not.